db/migrate/20181216204103_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,19 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[5.2]
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    # add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
+    add_index :users, :confirmation_token, unique: true
+    # User.reset_column_information # Need for some types of updates, but not for update_all.
+    # To avoid a short time window between running the migration and updating all existing
+    # users as confirmed, do the following
+    User.update_all confirmed_at: DateTime.now
```

Comment:
First of all do not use real model classes in your migrations. At some point you may rename or completely remove `User` class. Better write raw SQL or create "fake" models on the fly:
``` ruby
class UserStub < ActiveRecord::Base
  self.table_name = 'users'
end

def up
  UserStub.reset_column_information!
  UserStub.update_all(confirmed_at: DateTime.now)
end
```
Your case is quite simple, so `execute("update users set confirmed_at = now()")` should be enough.

Also prefer `AS:::TimeWithZone` over `Time/Date/DateTime`. `Time.zone.now` (or `Time.current`) is the right equivalent. If your app is in +12 TZ and your server's system time is in UTC (+0) `DateTime.now` may return a timestamp from the future.